src/content/peerconnection/audio/js/graph.js

```diff
@@ -0,0 +1,690 @@
+// Copyright (c) 2013 The Chromium Authors. All rights reserved.
```

Comment:
Sam, how should we deal with this copyright, which is slightly different than the webrtc-samples copyright?